public/javascripts/spacedeck_sections.js

```diff
@@ -2524,11 +2525,18 @@ var SpacedeckSections = {
     },
 
     show_toolbar_props: function() {
-      if (this.selection_metrics.count==0) return;
+      if (this.selection_metrics.count==0) {
+        this.toolbar_lock_in = false;
+        return;
+      }
       arts = this.selected_artifacts();
+      // check if selected artifacts are all from the same user
+      let userArts = true;
```

Comment:
nit: please rename to `same_user`, for example, to stay in line with other code style (even if idiosyncratic).